include/openspace/mission/mission.h

```diff
@@ -81,13 +90,43 @@ class MissionPhase {
      */
     const std::string& description() const;
 
+    /**
+     * Returns the associated image of this MissionPhase. If no image is associated, this
+     * string will be empty.
+     * \return The associated image of the MissionPhase or the empty string
```

Comment:
We'd have an extra empty
` *     `
line in here to separate the description from the return values